squib/_output/gitkeep.txt

```diff
@@ -0,0 +1 @@
+Keep this here so that Git knows to keep the _output directory on a fresh clone
```

Comment:
Ignore _output entirely, we don't want to commit what are essentially build artifacts to source control.